magento/magento2-page-builder
# Block preview in page builder broken for blocks that do not belong in the default strore

### Description (*)

If we insert in the PageBuilder editor a block which is configured for any store view other than default (or the first available if there is no default), the CMS block content will not be loaded in the preview. This happens because the environment emulation is run for the default store id and if the CMS block is configured for another store view, the html content will not be rendered for it

### Task


### Fixed Issues (if relevant)

1. https://github.com/magento/magento2/issues/36383


### Manual testing scenarios (*)

1. Have a magento instance with at least two stores
2. Create a cms block only visible in the store that is not the default store
3. Create a cms page in on of the two stores (doesn't matter which one)
4. Add the cms block you created, in the page through the page builder.
5. The previews should be displayed correctly


### Checklist
